services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return movies
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+        movies = movies.filter(actors__id__in=actors_ids)
+        return movies
```

Comment:
When both `genres_ids` and `actors_ids` are provided, the current logic applies two separate filters, which results in an 'AND' condition. If the intention is to filter movies that match any of the provided genres or actors, consider using `Q` objects to combine filters with an 'OR' condition.